kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/repository/FileRepositoryTest.java

```diff
@@ -120,7 +120,9 @@ public void testWatchServiceOnConfigFile() throws Exception {
         ContainerInfo containerInfo = new ContainerInfo("test1.0", "test", "org.kie:test:1.0");
         config.addContainerInfo(containerInfo);
 
-        File repositoryDirectory = new File("target" + File.separator + UUID.randomUUID().toString());
+        String fileDir = "target" + File.separator + UUID.randomUUID().toString();
```

Comment:
target is set in the before method as system property KieServerRouterConstants.ROUTER_REPOSITORY_DIR, you could use it here
https://github.com/kiegroup/droolsjbpm-integration/blob/69407ce98d7826e788d32a269f4d55157de6c7a7/kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/repository/FileRepositoryTest.java#L47

Reply:
indeed! done